packages/testing/src/MockProviders.tsx

```diff
@@ -55,7 +57,7 @@ export const MockProviders: React.FunctionComponent = ({ children }) => {
     <AuthProvider client={mockAuthClient} type="custom">
       <RedwoodApolloProvider useAuth={fakeUseAuth}>
         <UserRouterWithRoutes />
-        {children}
+        <LocationProvider>{children}</LocationProvider>
```

Comment:
Should we pass a mocked location here?

Reply:
I don't think that's needed, at least for now. You get a proper location back when using `useLocation()`. Here's an example screenshot. "/iframe.html" is from `const { pathname } = useLocation()`

![image](https://user-images.githubusercontent.com/30793/107098055-a7dc5e00-680e-11eb-8d6d-55e4746ac56b.png)